README.md

```diff
@@ -18,99 +22,137 @@ We have active, helpful communities on Twitter, Discord, and Telegram.
+
+> NOTE: The Steward CLI and Steward itself are distinct concepts in this
+> document. The Steward CLI is used to start both the `steward` and
+> `orchestrator` processes, while "Steward" refers specifically to the
+> `steward` process.
+
 ### Installation
 
-```bash 
+```bash
 # Create an installation directory
 mkdir install && cd install
 
-# Install Orchestrator
-wget https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.9/gorc && chmod +x * && sudo mv * /usr/bin
-
-# Install Geth
-wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.4-aa637fd3.tar.gz && tar -xvf geth-linux-amd64-1.10.4-aa637fd3.tar.gz && sudo mv geth-linux-amd64-1.10.4-aa637fd3/geth /usr/bin/geth && rm -rf geth-linux-amd64-1.10.4-aa637fd3*
+# Install Steward
+wget https://github.com/PeggyJV/steward/releases/latest/download/steward \
+    && chmod +x * \
+    && sudo mv * /usr/bin
 
 # Install Sommelier
-wget https://github.com/PeggyJV/sommelier/releases/download/v3.1.1/sommelier_3.1.1_linux_amd64.tar.gz && tar -xf sommelier_3.1.1_linux_amd64.tar.gz && sudo mv sommelier /usr/bin && rm -rf sommelier_3.1.1_linux_amd64* LICENSE README.md
+wget https://github.com/PeggyJV/sommelier/releases/download/v3.1.1/sommelier_3.1.1_linux_amd64.tar.gz \
+    && tar -xf sommelier_3.1.1_linux_amd64.tar.gz \
+    && sudo mv sommelier /usr/bin \
+    && rm -rf sommelier_3.1.1_linux_amd64* LICENSE README.md
 
-# Fetch systemd unit files
-wget https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/geth.service https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/gorc.service https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/sommelier.service
+# Fetch systemd unit file examples
+wget \
+    https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/sommelier.service \
+    https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/orchestrator.service \
+    https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/steward.service
 
 # Modify the unit files to fit your environment
-nano geth.service
-nano gorc.service
+nano orchestrator.service
+nano steward.service
 nano sommelier.service
 
 # And install them to systemd
-sudo mv geth.service /etc/systemd/system/geth.service && sudo mv gorc.service /etc/systemd/system/ && sudo mv sommelier.service /etc/systemd/system/ && sudo systemctl daemon-reload
-
-# Start geth
-sudo systemctl start geth && sudo journalctl -u geth -f
-
-# Init gorc configuration
-mkdir -p $HOME/gorc && cd $HOME/gorc
+sudo mv orchestrator.service /etc/systemd/system/ \
+    && sudo mv steward.service /etc/systemd/system/ \
+    && sudo mv sommelier.service /etc/systemd/system/ \
+    && sudo systemctl daemon-reload
+
+# Init steward/orchestrator configuration. Note that the steward and orchestrator processes share
+# much of the same configuration fields, so we share the config.toml for convenience.
+mkdir -p $HOME/steward && cd $HOME/steward
```

Comment:
The `config.toml` below is old and could use updating for the latest steward release. It doesn't point to keys for steward the correct way for example, and also overrides the default gas adjustment with a bad one.

Reply:
Updated